_posts/2019/2019-06-25-euruko-2019-conference-on-a-ship.md

```diff
@@ -0,0 +1,96 @@
+With ruby using **mRuby** and Lego he did a robot that can type a message.
+
+{% image_custom image="/assets/img/pages/blog/images/mruby.jpg" width="40" lightbox %}
+
+## Second day
+
+We started off with a list of possible cities for the next Euruko, amongst them also Melbourne.
+Yes, Australia!
+
+One talk taking my special attention that day was **Yes, I Test In Production... And So Should You**.
+[Charity Majors](https://twitter.com/mipsytipsy){:target="_blank"} explained us how we can find 80% of bugs having only a 20% effort by testing in production.
+She encouraged us to not be afraid about testing in production as we can find more bugs due to the production environment being different than staging.
+But sure, we shouldn't test in production only.
+
+{% image_custom image="/assets/img/pages/blog/images/realdata.jpg" width="40" lightbox %}
+
+[Yusuke Endoh](https://twitter.com/mametter){:target="_blank"} deepened more in the topic of **Ruby3** and we were able to see how types will work.
+[Jan Krutisch](https://twitter.com/halfbyte){:target="_blank"} gave us a master class of music, and he showed us how we can make music with ruby.
+It was interesting and exciting at the same time.
+
+Finally I want to highlight the last talk of the day: **The Past, Present, and Future of Rails at GitHub**.
+[Eileen M. Uchitelle](https://twitter.com/eileencodes){:target="_blank"} talked about her experience on a fork of rails to have a custom feature and her odyssey to update it.
+She highlighted that we need to invest in updating our apps.
+If we don't do that, in the future we can have a monster such as the legendary Hydra witch - for each bug that we can solve, there're 3 more.
+
+## Specials
+
+Each day we had a special talk: the first day [Melanie Keatley](https://twitter.com/Keatley){:target="_blank"} showed us her owns Pokemons in her talk **It's very effective; using Pokemon to catch all code smells**.
+The second day [Richard Schneeman](https://twitter.com/schneems){:target="_blank"} as [Marie Kondo](https://twitter.com/mariekondo){:target="_blank"} explained us how we can relocate the code in order to optimize our code.
+In this talk he used his dotter as rabbit dubing, that was awesome.
+
+## Lighting talks
+
+Miriam from Stripe talked about a gem called [**Sorbet**](https://sorbet.org/){:target="_blank"} that allows us to see the normal method definitions, and how these could be integrated in our VSCode.
+I swear that that was what most pleased me.
+I've been trying for months that my visual works as with TypeScript.
+Another incredible talk was from Norma Miller who was parsing in real time to transform every talk into text.
+She talked about the machine that she uses, and in that moment more or less she typed 350000 words and had a ratio of 99.65%.
+It was amazing to see how she works! 
+Anecdote: a volunteer tried to replace her, and she couldn't make any phrase during her talk.
```

Comment:
Who couldn't make any phrase? The volunteer or the one who actually presented?

Reply:
The voluntier, because she didn't know how it works